### ISSUE TITLE
Renderer text is not shown when downloading "pdf", "jpeg" and "pdf" files

### ISSUE THREAD

**Issue by jkriDK** [imclem/Highcharts-export-module-asp.net#30]
Hi! I have an issue related to "Highcharts-export-module-asp.net". As you maybe know it is possible to add your own text (renderer feature) to a chart like shown in the following jsfiddle:

http://jsfiddle.net/8y3d9rua/2/

and described here:

Docs:
•http://api.highcharts.com/highcharts#Renderer.text
•http://api.highcharts.com/highcharts#chart.events.load

Unfortunately the renderer text is not shown when downloading "pdf", "jpeg" and PDF file? The renderer text is included when printing or when downloading in svg format. So those two options are working. Any ideas for how to get the renderer text included also in pdf, jpeg or PDF files?

**Comment by imclem** [comment]
Hi!

This is what the module does:
- It gets a SVG file from an AJAX query made by Highcharts itselfs
- It uses SVG.dll to convert this SVG to bitmap
- It converts Bitmap to every possible format

You can try to download the "SVG" version and see if the text is shown ? (I assume that it is because you said it wasn't for PDF and jpeg and you haven't specified SVG).

If it is, then it's probably a problem with SVG.dll, in this case consider waiting a little bit @Tek4 is preparing a new release with a new version of SVG.dll.

If even SVG does not contain this text it's probalby Highcharts who does not render it to the SVG.

Hope this helps

**Comment by jkriDK** [comment]
Hi!

Thank you for your comment.:_)  And "yes" when downloading a chart in SVG format the text is shown. 

The problem with the missing text are related to PNG, JPEG and PDF file formats

**Comment by Tek4** [comment]
jkriDK,

The graphic on your screen or printer is rendered by your browser. The graphic exported by this project is rendered by SVG.dll, which is a SEPARATE component not under the control of this project. Most likely, SVG.dll is rendering the SVG code incorrectly. In some cases, Highcharts is/was the problem. In either case, your issue cannot be solved here.

On the other hand, within a few days or less, I will be updating SVG.dll to a new v.2.0.0 version which recently became available. I will post here when I push the update. Hopefully, the new SVG.dll will solve your issue; otherwise, you need to direct your issue to the author of the SVG.dll component.

**Comment by larssb** [comment]
Hi Tek4,

I'm also anxious for testing if v.2.0.0 helps resolve this issue. Are you getting near to updating the SVG.dll to v2 for this project :-) ?

Looking forward to hear from you....thank you very much.

**Comment by me-abhi** [comment]
Hi,

I am getting Overflow Error on this line
CreateSvgDocument().Draw().Save(
                seekableStream,
                ImageFormat.Png);

**Comment by markzolotoy** [comment]
I am bit confused by the product. Is it for a client side exporting? Or can convert the provided chart definition and data into an image? If later how can I do it? I am running: http://localhost:8001/default.aspx and getting an actual chart. What I want is an image of this chart without client exporting. 

Thanks

**Comment by imclem** [comment]
@markzolotoy no it is for server side exporting. It can convert the provided chart to different format, but the rendering is done on the server

**Comment by markzolotoy** [comment]
So, how do I convert a chart? Can you please show available commands? URL example? I am on a hot project, please help.
I am reading your reply again. So, when I select "download ...." from the dropdown menu the actual conversion is taking a place on the server, right? The UI is sending chart's SVG as a parameter. Is that how it works?

**Comment by markzolotoy** [comment]
Yeah... seems like it is working that way. What I need is to generate an image on the server from provided chart definition and the data. No SVG is available. Any idea about that? Also the generated image does not match the original chart.
Original
![image](https://user-images.githubusercontent.com/7363944/103666145-d75f2880-4f39-11eb-8824-61091428a15e.png)
Generated
![image](https://user-images.githubusercontent.com/7363944/103666245-feb5f580-4f39-11eb-94bb-63a58fbccd09.png)
